h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_large.py

```diff
@@ -8,67 +8,38 @@
 
 
 def test_parser_svmlight_column_skip():
-    # generate a big frame with all datatypes and save it to svmlight
```

Comment:
10k rows and 10 columns doesn't seem large to me

Should all the references to "big" be removed and test renamed (`large` classifier dropped)? The classifiers are important - they denote at what test group will the test run.

Reply:
@michalkurka

Good call.  Yes, the test can now be moved to normal and the test name will be: pyunit_PUBDEV_5705_drop_columns_parser_svmlight.py